pages/common/st-flash.md

```diff
@@ -0,0 +1,15 @@
+# st-flash
+
+> Flash binary files to STM32 ARM Cortex microcontrollers.
+
+- Read 4096 bytes of the firmware:
+
+`st-flash read {{firmware.bin}} 0x8000000 4096`
```

Comment:
Suggestion - "Read 4096 bytes of the firmware starting from 0x8000000". 4096 and 0x8000000 should be enclosed within {{}}. And put `.bin` out of {{}} as its compulsory to use binary files.